client/state/comments/actions.js

```diff
@@ -20,6 +20,8 @@ import {
 } from '../action-types';
 import { NUMBER_OF_COMMENTS_PER_FETCH } from './constants';
 
+export const requestComment = ( { siteId, commentId } ) => ( { siteId, commentId } );
```

Comment:
interesting thing here, all the tests pass because we short circuit the type -> action resolver...